### PR TITLE
Add text color (RGB)

### LINKED ISSUE #31: Add color for REX text

### PR DESCRIPTION
**Pull request by breiting**
Add color for the text block. RGB can now be specified using float values between 0 and 1.

resolved #31

### CODE REVIEW

File: doc/rex-spec-v1.md [hunk 1]
```diff
@@ -162,8 +162,14 @@ can be interpreted as 0 being the highest level. As data type we use 32bit for b
...
 #### DataType Text (1)
 
+This data type allows to position a (colored) text somewhere in space. The unit
+is again meters.
+
 | **size [bytes]** | **name**  | **type** | **description**                   |
 |------------------|-----------|----------|-----------------------------------|
+| 4                | red       | float    | red channel                       |
+| 4                | green     | float    | green channel                     |
+| 4                | blue      | float    | blue channel                      |
```
Comment (re-cheid) [inline doc/rex-spec-v1.md]: add another float for alpha (also at the lineset block)

File: src/rex-block-text.c [hunk 2]
```diff
@@ -27,8 +27,9 @@ uint8_t *rex_block_write_text (uint64_t id, struct rex_header *header, struct re
     uint16_t text_len = strlen (text->data);
 
     *sz = REX_BLOCK_HEADER_SIZE
-          + sizeof (float) * 3
-          + sizeof (float)
+          + sizeof (float) * 3 // color
```
Comment (re-cheid) [inline src/rex-block-text.c]: `sizeof (float) * 4` with alpha then

File: src/rex-block-text.h [hunk 3]
```diff
@@ -21,6 +21,9 @@
  *
  * | **size [bytes]** | **name**  | **type** | **description**                   |
  * |------------------|-----------|----------|-----------------------------------|
+ * | 4                | red       | float    | red channel                       |
+ * | 4                | green     | float    | green channel                     |
+ * | 4                | blue      | float    | blue channel                      |
```
Comment (re-cheid) [inline src/rex-block-text.h]: alpha

**Comment by breiting** [comment]
Done.

**Pull request merged**